src/ui/page/file/view.jsx

```diff
@@ -92,7 +92,9 @@ class FilePage extends React.Component<Props> {
     setViewed(uri);
   }
 
-  componentWillReceiveProps(nextProps: Props) {
+  // This is now marked as a react lecacy method:
+  // https://reactjs.org/docs/react-component.html#unsafe_componentwillmount
+  UNSAFE_componentWillReceiveProps(nextProps: Props) {
     const { fetchFileInfo, uri, setViewed } = this.props;
```

Comment:
This can just be updated to `componentDidUpdate`

`nextProps` becomes `prevProps`
and `this.props` references the new props